llvm/lib/Target/AIE/AIE2RegisterInfo.cpp

```diff
@@ -457,3 +457,32 @@ AIE2RegisterInfo::getCoveringSubRegs(const TargetRegisterClass &RC) const {
   }
   return Subregs;
 }
+bool AIE2RegisterInfo::isVecOrAccRegClass(const TargetRegisterClass *RC) const {
```

Comment:
Nit: We expect the pointer to be non-null, so it is better to use a reference.

Reply:
fixed

Reply:
I'll re-open the discussion so I remember to check after you push the changes :)

Reply:
should be fixed now. (I broke it in one of my earlier "fixups")